### ISSUE TITLE
Train ssd_mobilenet_v2_coco

### ISSUE THREAD

**Issue by PythonImageDeveloper** [tensorflow/models#3746]
Hi,
How we train the SSD_mobilenet_v2_coco ? i don't find it's respective check point. and why this is not possible for focal loss ?

**Comment by offbye** [comment]
I  have  tried to use a  pretrained  mobilenet v2 classify model  finetune, but after about 20000 steps, the model detected nothing,  any ideas ?

**Comment by zeynali** [comment]
@offbye , Where did you download the check point of mobilev2 ? please more explain about your work.

**Comment by seefun** [comment]
@offbye I meet the same problem. Haved you solved it?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
